docs/content/prometheus.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Monitoring the Ingress Controller Using Prometheus
+
+description: 
+weight: 100
+doctypes: ["concept"]
+toc: true
+---
+
+
+This doc is now available at https://docs.nginx.com/nginx-ingress-controller/logging-and-monitoring/prometheus/
```

Comment:
delete this doc and add an alias in the new file's frontmatter

Reply:
done